cluster/gce/gci/configure-helper.sh

```diff
@@ -656,7 +664,49 @@ contexts:
 EOF
   fi
 
-if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
+  if [[ -n "${WEBHOOK_CLIENT_KEY:-}" && -n "${WEBHOOK_CLIENT_CERT:-}" && -z "${GENERIC_ADMISSION_KUBECONFIG:-}" ]]; then
+    GENERIC_ADMISSION_KUBECONFIG="users:
+- name: >-
```

Comment:
What is `>-`?

Reply:
This ended up being the only way to get * as the value of the field in yaml reliably.
```
name: *
```
doesn't work because * is a special character in yaml
```
name: "*"
```
also didn't work because our yaml parser renders the quotes for some reason.
Should I leave a comment explaining what that means?

Reply:
> means that the value is a block of text and the - means to not include the newline at the end

Reply:
Please put what you said in a comment :)